### ISSUE TITLE
Issue Fix Request: App stalls on iOS devices after v2.14.13

### ISSUE THREAD

**Issue by Mad-Max-Sharecare** [zzorba/ArkhamCards#257]
Hi!
After upgrading to 2.14.13 version from the App Store, app no longer runs on some iOS devices - it hangs indefinitely on the Launch Screen.
The same happens in the latest v2.14.14 as well.

Tested on following devices:
iPhone SE 1st gen, iOS 12.4.1, 32 GB (MP822HN/A)
iPhone Xs, iOS 13.2.3, 64 GB (MT942LL/A)
iPad Air 2, iOS, 13.5.1, 16 GB (MH0W2LL/A)

The only device the app runs normally is:
iPhone 6+, iOS 12.4.7 16 GB (MGAL2LL/A)

Thanks in advance ;)
Thanks for the great app :)

**Comment by zzorba** [comment]
K, tried doing a full clean/rebuild and posted build 9.  Frustratingly, even that build seemed to work fine for me when I installed it myself on TestFlight on two different devices.

**Comment by zzorba** [comment]
Excellent!  I got this released wide last night, thank you so much for your help.

I believe the issue was actually mapping symbols onto emoji characters sets in my fonts (someone had been using my fonts for another project, and asked for this -- and I didn't realize it mattered).  This was months ago, but with the addition of the frost icon from edge of the earth, I regenerated the fonts for the first time, causing this issue to start cropping up.

Thank you so much for all your help and patience getting to the bottom of this with me.
